scene/3d/spatial.cpp

```diff
@@ -246,10 +246,7 @@ void Spatial::set_transform(const Transform &p_transform) {
 
 void Spatial::set_global_transform(const Transform &p_transform) {
 
-	Transform xform =
-			(data.parent && !data.toplevel_active) ?
-					  data.parent->get_global_transform().affine_inverse() * p_transform :
-					  p_transform;
+	Transform xform = (data.parent && !data.toplevel_active) ? data.parent->get_global_transform().affine_inverse() * p_transform : p_transform;
 
```

Comment:
Gave up here, clang format seems to consistently produce a result which fails the CI checks for this. Clang format seems to give 6 tabs and 2 spaces, whereas the static checks want 5 tabs and 2 spaces?

Reply:
You might need to upgrade to clang-format 11 to get the same results as the CI. But yeah, clang-format seems to have troubles handling long ternary operators in a consistent way.

Reply:
Ah I'll try the clang-format upgrade, see if it helps! :+1:

Reply:
clang-format-11 gives 8 tabs. Maybe something they haven't quite got sorted yet! :grin: